packages/react-components/src/components/chart/legend/tanstackLegend.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { DataStream } from '@iot-app-kit/core';
+import { ColumnDef } from '@tanstack/react-table';
+
+import { SeriesOption } from 'echarts';
+import useChartsLegendTanstack from './useChartsLegendTanstack';
+
+import { InternalGraphicComponentGroupOption } from '../trendCursor/types';
+import { TanstackTable } from './tanstackTable';
+import './legend.css';
```

Comment:
nit: do you need this CSS file imported ?

Reply:
updated